### PR TITLE
[PowerMock] LPS-152436 Remove powermock usages (Misc) part 4

### PR DESCRIPTION
**Pull request by dantewang**
@tinatian

### CODE REVIEW

**Comment by dantewang** [comment]
Original description: https://github.com/dantewang/liferay-portal/pull/2217#issue-1226235171

> This PR is to remove powermock usages (Misc) part 4. Thanks for checking.
>
> https://issues.liferay.com/browse/LPS-152436
> 
> apps.blogs
> apps.dynamic-data-mapping-form-values-factory
> apps.adaptive-media
> apps.data-engine-rest-api
> apps.dynamic....dynamic-data-mapping-form-field-type

After this PR, only the following tests are left with PowerMock usages. These tests mock static methods which has relatively complicated logic.

SearchResultUtilDLFileEntryTest | apps.document-library-service
-- | --
DefaultBearerTokenProviderTest | apps.oauth2-provider
DocumentHTMLProcessorTest | apps.document-library-document-conversion
  |  
BaseJSONWebServiceTestCase | portal-impl
JSONWebServiceInvokerTest | portal-impl
JSONWebServiceTest | portal-impl

**Comment by JanisZhang** [comment]
Hi @catarineaguiar,
Could you please help me scan test results. Thx!

Regards.
Janis.

File: modules/apps/adaptive-media/adaptive-media-image-service/src/test/java/com/liferay/adaptive/media/image/internal/exportimport/content/processor/AMImageHTMLExportImportContentProcessorTest.java [hunk 1]
```diff
@@ -151,9 +152,10 @@ public void testExportImportContentWithMultipleReferences()
 
 		Assert.assertEquals(
 			StringBundler.concat(
-				prefix, "<img src=\"", urlFileEntry1, "\" data-fileentryid=\"",
-				_FILE_ENTRY_ID_1, "\" />", infix, "<img src=\"", urlFileEntry2,
-				"\" data-fileentryid=\"", _FILE_ENTRY_ID_2, "\" />", suffix),
+				prefix, "<img src=\"", urlFileEntry1,
```
[the remark this answers is not in the document]
Comment (JanisZhang) [reply modules/apps/adaptive-media/adaptive-media-image-service/src/test/java/com/liferay/adaptive/media/image/internal/exportimport/content/processor/AMImageHTMLExportImportContentProcessorTest.java:155]: Hi @tinatian, 
The original test is to mockstatic ExportImportPathUtil ,and  it cannot be mocked by Mockito, so we use this util directly and change the expected to cope with the returned value.

Or we can leave it for now until we upgrade the Mockito.  cc @dantewang

**Comment by JanisZhang** [comment]
ci:close

**Pull request closed without merging**